openmdao/visualization/htmlpp.py

```diff
@@ -94,7 +93,7 @@ class HtmlPreprocessor():
     Nothing is written until every directive has been successfully processed.
     """
 
-    def __init__(self, start_filename: str, output_filename: str, start_path: str = None,
+    def __init__(self, start_filename: str, output_filename: str, search_path=[],
```

Comment:
NOTE: a mutable default value is not recommended best-practice
https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/